From c991820ef19cc57b7c83dfa86f195d9ed12ec501 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Thu, 7 Mar 2019 16:55:53 +0100
Subject: [PATCH] Support early loading of keymap if kbd is installed

Early loading of the keymap with busybox was never supported and would
require modifying the save-keymaps services as well. Since no one
complained that it doesn't work with busybox so far just make it work
with kbd for now.
---
 sh/init-early.sh.Linux.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sh/init-early.sh.Linux.in b/sh/init-early.sh.Linux.in
index b7082a02..777cbda9 100644
--- a/sh/init-early.sh.Linux.in
+++ b/sh/init-early.sh.Linux.in
@@ -48,8 +48,8 @@ if service_present "$RC_DEFAULTLEVEL" consolefont ||
 fi
 
 # Try and set a keyboard map as early as possible
-if service_present "$RC_DEFAULTLEVEL" keymaps ||
-   service_present "$RC_BOOTLEVEL" keymaps; then
+if service_present "$RC_DEFAULTLEVEL" loadkeys ||
+   service_present "$RC_BOOTLEVEL" loadkeys; then
 	kbd_mode $kmode -C "$CONSOLE" 2>/dev/null
 	if [ -r "$RC_LIBEXECDIR"/console/keymap ]; then
 		loadkeys -q "$RC_LIBEXECDIR"/console/keymap 2>/dev/null
